docs/Analytics.md

```diff
@@ -14,14 +14,14 @@ Homebrew's anonymous user and event data have a 14 month retention period. This
 ## What?
 Homebrew's analytics record some shared information for every event:
 
-- The Homebrew user agent, e.g. `Homebrew/0.9.9 (Macintosh; Intel macOS 10.12.0) curl/7.43.0`
-- The Google Analytics version, i.e. `1` (https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#v)
-- The Homebrew analytics tracking ID, e.g. `UA-75654628-1` (https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#tid)
-- A Homebrew analytics user ID, e.g. `1BAB65CC-FE7F-4D8C-AB45-B7DB5A6BA9CB`. This is generated by `uuidgen` and stored in the repository-specific Git configuration variable `homebrew.analyticsuuid` within `$(brew --repository)/.git/config`. This does not allow us to track individual users but does enable us to accurately measure user counts vs. event counts (https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#cid)
-- If the Google Analytics anonymous IP setting is enabled, i.e. `1` (https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#aip)
-- The Homebrew application name, e.g. `Homebrew` (https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#an)
-- The Homebrew application version, e.g. `0.9.9` (https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#av)
-- The Homebrew analytics hit type, e.g. `event` (https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#t)
+- The Homebrew user agent, e.g. `Homebrew/0.9.9 (Macintosh; Intel macOS 10.12.0) curl/7.43.0`.
```

Comment:
Nit: As we're changing this line to add a full stop, can we update the user agent string to something that's not ancient? 💀

Reply:
Interestingly, the user agent is listed as OS X, not macOS:

```console
% brew ruby -e 'puts HOMEBREW_USER_AGENT_CURL'
Homebrew/2.5.0-8-gac8282b (Macintosh; Intel Mac OS X 10.15.6) curl/7.64.1
```

Reply:
Apparently this is intentional: https://github.com/Homebrew/brew/blob/master/Library/Homebrew/brew.sh#L154-L155, and on my Catalina machine, yep, Safari still reports "Mac OS X".